### PR TITLE
Add youtube links to module pages and pdf outputs

### PR DESCRIPTION
**Pull request by sumandari**
This PR refers to #1232. It does:
- add video link in PDF output
- embed video on web form 

changelog_1232_youtubelink

### CODE REVIEW

File: django_project/lesson/templates/worksheet/print.html [hunk 1]
```diff
@@ -213,6 +213,11 @@ <h2>{% trans 'Check your knowledge:' %}</h2>
         </div>
         <h2>{% trans 'Further reading:'%}</h2>
         <ul>
+            {% if worksheet.youtube_link %}
+                <li>
+                    <a target="_blank" href="{{ worksheet.youtube_link }}">module video tutorial</a>
```
Comment (flippant-heron) [inline django_project/lesson/templates/worksheet/print.html]: Can we change the casing on this to "Module video tutorial" and also expose it as a description, with the full link represented the same way other elements within the "Further reading" section are

**Review by zacharlie — CHANGES_REQUESTED** [review]
Looking good, I would just like to see some minor changes to the PDF link 👍

**Comment by sumandari** [comment]
@zacharlie, I updated the PR, it looks like:

1224_youtubelink

**Review by zacharlie — APPROVED** [review]
Awesome thanks! 👍

File: django_project/lesson/templates/worksheet/print.html (hunk 1 above)
Comment (sumandari) [reply django_project/lesson/templates/worksheet/print.html]: @zacharlie, I updated the PR, it looks like:

1224_youtubelink

**Comment by timlinux** [comment]
Good job thanks @sumandari

**Pull request merged**